src/app/blog/[slug]/page.tsx

```
@@ -0,0 +1,224 @@
+              </div>
+              <div className="flex flex-center gap-2">
+                <ClockIcon className="w-5 h-5" />
+                <span>{post.reading_time} min read</span>
+              </div>
+              {primaryAuthor && (
+                <div className="flex flex-center gap-2">
+                  {primaryAuthor.profile_image && (
+                    // eslint-disable-next-line @next/next/no-img-element
+                    <img
+                      src={primaryAuthor.profile_image}
+                      alt={`${primaryAuthor.name}'s profile picture`}
```

Biome (2.1.2)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)